Configurations.md

```diff
@@ -1306,6 +1306,15 @@ If you want to ignore every file under the directory where you put your rustfmt.
 ignore = ["/"]
 ```
 
+## `ignore_missing_submod`
+
+Ignore missing submodule error.
+By default, missing modules will lead to module not found errors and rustfmt won't format anything.
+
+- **Default value**: `false`
+- **Possible values**: `true`, `false`
```

Comment:
On thing the team has discussed is moving away from a binary error model. I think if we land this, I'd like to change from `true` and `false` to something like `Warn`, `Error`, `Ignore`. With that in mind I think it a more appropriate name would be something like `report_missing_submod`. Let me know if you can come up with anything better!

Reply:
Update Configuration.md on this commit. https://github.com/rust-lang/rustfmt/pull/5611/commits/4bb045340276b8fade94dd822dbb34ba40c4ed49